.github/workflows/cli-test-homebrew.yml

```diff
@@ -17,13 +17,17 @@ jobs:
         uses: actions/setup-node@v2
         with:
          node-version: ${{ matrix.node-version }}
+      - name: Check Out Repo
+        uses: actions/checkout@v2
       -  name: Global - Testing installation of Twilio CLI (1)
          run: |
-          brew tap twilio/brew && brew install twilio@${{ github.event.inputs.version }}
-          node test/asserts/validate_tests.js
+          brew tap twilio/brew && brew install twilio
+          chmod +x ./.github/scripts/cmds-to-test.sh
+          ./.github/scripts/cmds-to-test.sh
       - name: Global - Testing upgrade Twilio CLI (2)
         run: |
-          brew tap twilio/brew && brew install twilio@${{ github.event.inputs.version }}
+          brew tap twilio/brew && brew install twilio
           brew upgrade twilio
-          node test/asserts/validate_tests.js
+          chmod +x ./.github/scripts/cmds-to-test.sh
+          ./.github/scripts/cmds-to-test.sh
```

Comment:
[Q] Are we sure here that the action node wouldn't be re-used with any stale caches? 
Asking since if the container is reused internally by Github Actions with a pre-installed CLI, there might be a conflict and we might not be testing the correct version. 
A very edge case- But could we add a sanity step before installation to test `twilio --version` that it errors out? Do you think it's worth the effort?

Reply:
I was worried about the caching too, found out that in order to cache dependencies, actions/cache should be explicitly called out. Also, GH stores caches for ~7 days, so we are covered for the most of it. I did test with this edge case but didn't add it in. 
I'll look at this as an if-else situation so that it errors out with an appropriate error msg.